pkgs/development/libraries/c-ares/default.nix

```
@@ -50,6 +48,10 @@ stdenv.mkDerivation rec {
     curl = (curl.override { c-aresSupport = true; }).tests.withCheck;
   };
 
+  preFixup = lib.optionalString withCMake ''
+    sed -i 's,''${prefix}/,,g' $out/lib/pkgconfig/libcares.pc
```

```suggestion
    substituteInPlace $out/lib/pkgconfig/libcares.pc --replace-fail \''${prefix} ""
```
Not 100% certain the dollar is escaped correctly

---

applied recommendation, with an appended '/' to prevent '//' from showing up in the pkgconfig file